package.json

```diff
@@ -25,6 +25,7 @@
     "babel-register": "^6.24.0",
     "babelify": "^7.3.0",
     "browserify": "^13.3.0",
+    "check-dependencies": "^1.1.0",
```

Comment:
Checks for missing or out-of-date dependencies.  If things need fixing it tells the user to run `npm install`.